.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
is this supposed to be empty?

Reply:
weirdly, yes it is actually

It tells the IDE to use prettier I think?